src/physics/initial_conditions/initial_condition.cpp

```diff
@@ -230,6 +230,8 @@ InitialConditionFactory<dim,nstate, real>::create_InitialConditionFunction(
         }
     } else if (flow_type == FlowCaseEnum::periodic_1D_unsteady){
         if constexpr (dim==1 && nstate==dim)  return std::make_shared<InitialConditionFunction_1DSine<dim,nstate,real> > ();
+    } else if (flow_type == FlowCaseEnum::sshock) {
+        if constexpr (dim==2 && nstate==1)  return std::make_shared<InitialConditionFunction_Zero<dim,nstate,real> > ();
```

Comment:
the flow type is called `sshock` but is initialized as zero? maybe rename this to `zero`

Reply:
Not sure about that, because `sshock` is the manufactured flow solution (i.e. final converged solution) and `zero` is the initial condition.

Reply:
So the steady state flow is driven by boundary conditions?

Reply:
Yes (and the solution at boundary is set as the manufactured solution).